cli/command/stack/remove.go

```diff
@@ -38,7 +38,10 @@ func runRemove(dockerCli command.Cli, opts removeOptions) error {
 	namespaces := opts.namespaces
 	client := dockerCli.Client()
 	ctx := context.Background()
-
+	version, err := client.ServerVersion(ctx)
```

Comment:
Oh, I know what I did 🤦‍♂️ - yes this works, thanks! Adding some additional test-cases for different versions now